lib/presentation/widgets/logo_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class LogoWidget extends StatelessWidget {
+  const LogoWidget({Key? key}) : super(key: key);
+
+  static const logoText = 'WaterApp';
+  static const logoImagePath = "assets/images/logo.svg";
+  static const logoHeight = 18.75;
```

Comment:
do u think, we need to be such precise with sizes?

Reply:
I'm not sure, but I'm not thinking it`s bad